src/average_coverage/core.clj

```diff
@@ -49,5 +49,5 @@
                      (map #(vec (map :percentage_coverage %))))
         mean-coverage (map #(double (/ (reduce + %)
                                       (count %))) lengths)]
-    (zipmap genes mean-coverage)))
+    (spit "average_coverage.txt" (zipmap genes mean-coverage))))
```

Comment:
I'd make the main take 2 args so you can do in and out files and not hard code the out